Cannot find name 'SnapshotStateOptions'

Getting this error when running `tsc`, with `"skipLibCheck": false`

```
node_modules/mocha-chai-jest-snapshot/dist/utils/jest-test-result-helper.d.ts:14:58 - error TS2304: Cannot find name 'SnapshotStateOptions'.

14 export declare const makeEmptySnapshotSummary: (options: SnapshotStateOptions) => SnapshotSummary;
                                                            ~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/mocha-chai-jest-snapshot/dist/utils/jest-test-result-helper.d.ts:14
```

--- a/src/utils/jest-test-result-helper.ts
+++ b/src/utils/jest-test-result-helper.ts
@@ -10,7 +10,8 @@ import {
   makeEmptyAggregatedTestResult,
 } from "@jest/test-result";
 import { SnapshotState } from "jest-snapshot";
-import { SnapshotStateOptions } from "jest-snapshot/build/State";
+
+type SnapshotStateOptions = ConstructorParameters<typeof SnapshotState>[1];
 
 type SnapshotResult = TestResult["snapshot"];
 

